x/monitoringc/types/errors.go

```diff
@@ -8,7 +8,11 @@ import (
 
 // x/monitoringc module sentinel errors
 var (
-	ErrSample               = sdkerrors.Register(ModuleName, 1100, "sample error")
-	ErrInvalidPacketTimeout = sdkerrors.Register(ModuleName, 1500, "invalid packet timeout")
-	ErrInvalidVersion       = sdkerrors.Register(ModuleName, 1501, "invalid version")
+	ErrInvalidPacketTimeout    = sdkerrors.Register(ModuleName, 1, "invalid packet timeout")
```

Comment:
As I know code 1 is reserved for internal usage. Error codes declaration must start with 2.

Reply:
Good point👍